### ISSUE TITLE
RSC 1.8.8 pin_versions created = NA

### ISSUE THREAD

**Issue by SaJaToGu** [rstudio/pins-r#591]
When list pin versions from RSC 1.8.8 the created date is NA.
In the RSC contend history function the dates can be seen.

`pins::board_rsconnect() %>% pins::pin_versions(xxx)
Connecting to RSC 1.8.8 at <https://xxx>
# A tibble: 6 × 4
  version created active   size
  <chr>   <dttm>  <lgl>   <dbl>
1 71093   NA      TRUE   334178
2 71092   NA      FALSE  334176
3 70990   NA      FALSE  337920
4 70980   NA      FALSE  337917
5 70962   NA      FALSE  337945
6 70955   NA      FALSE  337938`

**Comment by joranE** [comment]
I just noticed the same issue. The problem seems to stem from the internal function `parse_8601_compact()`, which assumes a datetime format of `"%Y%m%dT%H%M"` but what I see in the json object being operated on in `rsc_connect_versions` is `"2022-04-13T15:06:32Z"`, for example.

A very crude workaround is of course to pluck out the creation date manually from what's returned from `pin_meta`.

**Comment by juliasilge** [comment]
This has been fixed in #623 🎉

``` r
library(pins)
b <- board_rsconnect()
#> Connecting to RSC 2022.07.0 at <https://colorado.rstudio.com/rsc>
b %>% pin_versions("julia.silge/superbowl_rf")
#> # A tibble: 15 × 4
#>    version created             active   size
#>    <chr>   <dttm>              <lgl>   <dbl>
#>  1 60376   2022-08-12 16:11:00 TRUE    85667
#>  2 60232   2022-08-10 11:12:00 FALSE   85067
#>  3 58318   2022-06-29 09:57:00 FALSE   85206
#>  4 57404   2022-06-08 15:22:00 FALSE   88120
#>  5 57365   2022-06-07 13:41:00 FALSE   88185
#>  6 57364   2022-06-07 13:35:00 FALSE   87879
#>  7 57309   2022-06-06 09:17:00 FALSE   88767
#>  8 57291   2022-06-05 19:33:00 FALSE   88709
#>  9 55426   2022-04-19 13:20:00 FALSE  152018
#> 10 55158   2022-04-12 12:18:00 FALSE   89828
#> 11 54892   2022-04-04 09:14:00 FALSE   88961
#> 12 54422   2022-03-21 10:11:00 FALSE   88342
#> 13 54416   2022-03-21 09:20:00 FALSE   87386
#> 14 54302   2022-03-16 14:40:00 FALSE   87829
#> 15 53963   2022-03-08 13:34:00 FALSE   87735
```

<sup>Created on 2022-08-16 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>
